### PR TITLE
Improve wording for the active contracts service description

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
This should eliminate sources of confusion as mentioned here: https://discuss.daml.com/t/offset-is-null-while-getting-active-contracts/3118

changelog_begin
[Docs] Improved description of the purpose and usage of the active contracts service
changelog_end

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/app-dev/services.rst [hunk 1]
```diff
@@ -157,9 +157,9 @@ You can get these included in requests related to Transactions by setting the ``
 Active contracts service
 ========================
 
-Use the **active contracts service** to obtain a party-specific view of all contracts currently active on the ledger.
+Use the **active contracts service** to obtain a party-specific view of all contracts that are active on the ledger at the time of the request.
 
-The active contracts service returns the current contract set as a set of created events that would re-create the state being reported. Each created event has a ledger offset where it occurs. You can infer the ledger offset of the contract set from the ledger offset of the last event you receive.
+The active contracts service returns its response as a stream of batches of the created events that would re-create the state being reported (the size of these batches is left to the ledger implementation). After all contracts have been reported, one final message includes the offset at which this was the state. This offset can be used to subscribe to the "flat transactions" stream to keep a consistent view of the active contract set without querying the active contract service further.
```
Comment (cocreature) [inline docs/source/app-dev/services.rst:162]: The description "after all contracts have been reported" makes it sound like we guarantee that the message with the offset has no contracts itself. Is that actually a guarantee we want to make?

Comment (stefanobaghino-da) [reply docs/source/app-dev/services.rst]: Addressed by https://github.com/digital-asset/daml/pull/10880/commits/f98677d5f45b590dabe3da3948be382d79a810e1

File: docs/source/app-dev/services.rst [hunk 2]
```diff
@@ -157,9 +157,9 @@ You can get these included in requests related to Transactions by setting the ``
 Active contracts service
 ========================
 
-Use the **active contracts service** to obtain a party-specific view of all contracts currently active on the ledger.
+Use the **active contracts service** to obtain a party-specific view of all contracts that are active on the ledger at the time of the request.
 
-The active contracts service returns the current contract set as a set of created events that would re-create the state being reported. Each created event has a ledger offset where it occurs. You can infer the ledger offset of the contract set from the ledger offset of the last event you receive.
+The active contracts service returns its response as a stream of batches of the created events that would re-create the state being reported (the size of these batches is left to the ledger implementation). As part of the last message message, the offset at which which the reported active contract set was valid is included. This offset can be used to subscribe to the "flat transactions" stream to keep a consistent view of the active contract set without querying the active contract service further.
```
Comment (stefanobaghino-da) [inline docs/source/app-dev/services.rst:162]:
```suggestion
The active contracts service returns its response as a stream of batches of the created events that would re-create the state being reported (the size of these batches is left to the ledger implementation). As part of the last message message, the offset at which the reported active contract set was valid is included. This offset can be used to subscribe to the "flat transactions" stream to keep a consistent view of the active contract set without querying the active contract service further.
```

**Review by cocreature — APPROVED** [review]
Thanks!

**Pull request merged**